index.html

```diff
@@ -11,51 +11,183 @@
 </head>
 
 <body>
-  <header class="header">
-    <a href="">
-      <p class="welcome"> <strong>Welcome</strong></p>
-    </a>
-    <div class="vector_container">
+  <div class="page1Container">
+    <header class="header">
       <a href="">
-        <svg class="vectSvg vect1">
-          <rect class="vector" height="2" width="16" />
-        </svg>
-      </a>
-      <a href="">
-        <svg class="vectSvg vect2">
-          <rect class="vector" height="2" width="16" />
-        </svg>
-      </a>
-      <a href="">
-        <svg class="vectSvg vect3">
-          <rect class="vector" height="2" width="16" />
-        </svg>
+        <p class="welcome"> <strong>Welcome</strong></p>
       </a>
+      <div class="vector_container">
+        <a href="">
+          <svg class="vectSvg vect1">
+            <rect class="vector" height="2" width="16" />
+          </svg>
+        </a>
+        <a href="">
+          <svg class="vectSvg vect2">
+            <rect class="vector" height="2" width="16" />
+          </svg>
+        </a>
+        <a href="">
+          <svg class="vectSvg vect3">
+            <rect class="vector" height="2" width="16" />
+          </svg>
+        </a>
+      </div>
+    </header>
+    <section class="mainSection">
+      <article class="pageSection">
+        <h1 class="stText">
+          <b>Hey there. I’m Mariane</b>
+        </h1>
+        <h1 class="ndText">
+          <b>I’m a software developer</b>
+        </h1>
+      </article>
+      <article class="pageSection">
+        <p class="text">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </article>
+      <ul class="pageSection socialMedia">
+        <li> <a href="#" class="smIcom"><img src="images/twitter.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcom"><img src="images/Linkedin_icon.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcom"><img src="images/medium.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcom"><img src="images/github.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcom"><img src="images/angellist.svg" alt=""></a> </li>
+      </ul>
+      <div class="nextPageIcon">
+        <a href=""><img src="images/Disabled.svg" alt=""></a>
+      </div>
+    </section>
+  </div>
+  <div class="workTitle">
+    <h1 id="h1WorkTitle">My Recent Works</h1>
```

Comment:
- [ ] please its recommenced to follow HTML semantic code. So you should only have 1 `h1` tag per file.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: